### ISSUE TITLE
It's blinking in the opera browser

### ISSUE THREAD

**Issue by proton** [pisi/Reel#151]
You can test this page: http://vsestulya.ru/3d/i.php?model=tiesto
The image starts to blink after 5-10 seconds after the start.

You can downloand opera at this page: http://www.opera.com/

**Comment by pisi** [comment]
I know Opera and [test](/pisi/Reel/tree/master/TESTS.markdown#opera) against it regularly. Opera only doesn't support custom cursors, otherwise it's just fine.

I believe your problem is quite hinted by Opera's Connection tab of the built-in developer tools, which says:

> 81 requests, 13.06 MB

Dude, what a bonkers number! This all has to first pass through the network and then most of it passes through rendering many times per second. Opera may have problems to keep up with such extreme data load.

You should first **optimize your images for web**. Size around 180 kb per JPG 800x533 frame is fairly high for being tossed around at 36 Hz. It basically is max quality, which you don't need on the web. Your images can easily survive 70% quality, which is virtually un-noticable by naked eye and brings the size down to 45 kb per frame - that's 75% savings! That should make Opera and subsequently all your customers a bit happier ;)

**Comment by proton** [comment]
Thanks for advice. Images are optimized, it's still blinking :(

**Comment by pisi** [comment]
Frankly, I'm still not sure. It is the first time I encountered this kind of error.

I took your source, stripped everything else out of it and tried some stuff. What did help me to get rid of the blinking was using just every other image of your 72 images:

``` js
    images:      'clips/tiesto/##.jpg|1..72|2'
```

Not really pleasant "solution", I must admit. I am quite sure, that using a bit smaller image dimensions would help you much more. 

I really do guess that Opera may not be happy with redrawing image this big in this pace. The cost of Reel's operation (`$.reel.cost`) is around 3 ms per tick from the available 27.5 ms dictated by the default 36 Hz tempo, so that leaves quite a room for rendering I think. But, every JPG image has to be first decompressed into memory and ends up taking up a significantly larger space, than its actual file size. Have you maybe tried using smaller images? Or did you try progressive JPEGs?

**Comment by proton** [comment]
No, I didn't trying anything.
I added hack for opera, thanks.

``` javascript
if(jQuery.browser.opera)
{
    $('#image').reel({
        cw:          true,
        speed:       0.2,
        velocity:    2,
        images:      'clips/<?php echo $model?>/##.jpg|1..72|2'
    });
}
```
